src/ButtonToggle.js

```diff
@@ -13,12 +13,8 @@ const propTypes = {
   cssModule: PropTypes.object,
 };
 
-const defaultProps = {
-  defaultValue: false,
-};
-
 function ButtonToggle(props) {
-  const [toggled, setToggled] = useState(props.defaultValue);
+  const [toggled, setToggled] = useState(props.defaultValue || false);
```

Comment:
Would the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) be a better choice here?